src/styles/export.css

```diff
@@ -30,6 +30,13 @@
   }
 }
 
+/**
+ * Add text cursor on placeholder 
+ */
+.ce-header[contentEditable=true][data-placeholder]:empty::before {
```

Comment:
`ce-header` is a hardcode. You need to add `cursor` to the editor's element, not a plugin

Reply:
I think the problem only exists with the Header tool itself. I think @gohabereg is right that it should be moved to the Header tool itself.